change/@ni-nimble-components-f7f47d9d-f5d5-4ad0-b573-a1ab40586bfd.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Angular doesn't have a shared base class for mappings and probably should
Blazor does have a shared base class and should be updated to move Text out

Reply:
Created a tech debt task for this: https://github.com/ni/nimble/issues/1666